scrap_zenodo.py

```diff
@@ -34,14 +39,124 @@ def search_zenodo(page=1, hits_per_page=10, year=2016):
     return response.json()
 
 
-resp_json = search_zenodo(hits_per_page=100, year=2017)
-total_hits = resp_json["hits"]["total"]
-print(f'Number of hits: {total_hits}')
+# resp_json = search_zenodo(hits_per_page=100, year=2017)
+# total_hits = resp_json["hits"]["total"]
+# print(f'Number of hits: {total_hits}')
 
 
 # Record example
 
 response = requests.get("https://zenodo.org/api/records/53887",
                         params={"access_token": ZENODO_TOKEN})
 resp_json = response.json()
-print(resp_json)
+
+
+# print(resp_json)
+
+
+# Query: resource_type.type:"dataset" AND filetype:"tpr"
+# on the Zenodo website, we find 483 datasets.
+
+def search_tpr_zenodo(page=1, hits_per_page=10, year=2016):
```

Comment:
Could you please remove the `year` argument since it is not used anymore?

Reply:
Could you also make you function more generic to search for any filetype (trp, gro, mdp...). The filetype will be provided as an argument to the function. 
You could rename this function `search_zenodo_by_filetype` or something similar.